ui/pages/Project/selectors.js

```diff
@@ -769,8 +772,14 @@ export const getPageHeaderBreadcrumbIdSections = createSelector(
         content: `Family: ${family.displayName || ''}`,
         link: `/project/${project.projectGuid}/family_page/${family.familyGuid}`,
       }]
-      if (match.params.breadcrumbIdSection) {
-        breadcrumbIdSections.push({ content: snakecaseToTitlecase(match.params.breadcrumbIdSection), link: match.url })
+      const { breadcrumbIdSection } = match.params
+      if (breadcrumbIdSection) {
+        if (breadcrumbIdSection.startsWith('rnaseq_results/')) {
+          const individualId = individualsByGuid[breadcrumbIdSection.split('/')[1]]?.individualId || ''
+          breadcrumbIdSections.push({ content: `RNAseq: ${individualId}` })
+        } else {
+          breadcrumbIdSections.push({ content: snakecaseToTitlecase(breadcrumbIdSection), link: match.url })
+        }
```

Comment:
You also need to update the project `PageHeader` component to not show the family description on this page